src/shaders/line_sdf.vertex.glsl

```diff
@@ -13,12 +13,11 @@ attribute float a_linesofar;
 uniform mat4 u_matrix;
 uniform mediump float u_ratio;
 uniform lowp float u_device_pixel_ratio;
-uniform vec2 u_patternscale_a;
-uniform float u_tex_y_a;
-uniform vec2 u_patternscale_b;
-uniform float u_tex_y_b;
 uniform vec2 u_units_to_pixels;
 
+uniform vec2 u_texsize;
+uniform mediump vec3 u_scale;
```

Comment:
Could we change  `#define scale 0.015873016` (Line 7 here) to e.g. `SCALE` or, rename this uniform? It is related to metal shader compiling internals - u_scale gets transpiled to `u_.scale`, and there is name collision with constant `scale` `"error: '' :  syntax error, unexpected FLOATCONSTANT, expecting IDENTIFIER>"`.